src/request-utils.js

```diff
@@ -0,0 +1,51 @@
+import videojs from 'video.js';
+
+/**
+ * Requests all of the urls provided, then calls back.
+ *
+ * @param {string[]} urls
+ *        An array of urls
+ * @param {function(Object, Object)} callback
+ *        Callback function with error and object containing url to response text entries
+ */
+export const requestAll = (urls, callback) => {
+  let requestsRemaining = urls.length;
```

Comment:
should we make sure urls is unique here? to prevent requesting the same file twice.